openff/toolkit/topology/molecule.py

```diff
@@ -3275,6 +3275,147 @@ def generate_conformers(
                 )
             )
 
+    def _make_carboxylic_acids_cis(self):
+        """
+        Rotate dihedral angle of any conformers with trans COOH groups so they are cis
+
+        Carboxylic acid groups almost always exist in nature in the cis conformation,
+        with the hydrogen atom in between the two oxygen atoms::
+
+                  O----H
+                 /
+                /
+               /
+            --C
+               \\
+                \\
+                  O
+
+        However, the OpenEye toolkit frequently produces carboxylic acid geometries
+        in the unrealistic trans conformation::
+
+             H----O
+                 /
+                /
+               /
+            --C
+               \\
+                \\
+                  O
+
+        This method converts all conformers in the Molecule with the trans conformation
+        into the corresponding cis conformer by rotating the OH bond around the CO bond
+        by 180 degrees. Carboxylic acids that are already cis are unchanged. Carboxylic
+        acid groups are considered cis if their O-C-O-H dihedral angle is acute.
+        """
+
+        # Return early if there are no conformers
+        if not self._conformers:
+            return
+
+        # Convert all conformers into one big array
+        conformers = np.asarray(
+            [q.value_in_unit(unit.angstrom) for q in self._conformers]
+        )
+
+        # Pull out the coordinates of all carboxylic acid groups into cooh_xyz
+        # cooh_xyz is an array with shape (n_cooh_groups, n_conformers, 4, 3)
+        cooh_indices = self.chemical_environment_matches("[C:2]([O:3][H:4])=[O:1]")
```

Comment:
(not blocking) In other circumstances we'd want to pass in a toolkit registry here, and have the method take a toolkit registry as input. But since this is a private method I don't think it's necessary